Wrapping/Python/Examples/use_case_02.py

```diff
@@ -36,6 +45,35 @@ def import_csv_xyz_vertices_2(file_path: str):
     dca.addOrReplaceDataContainer(dc)
     # Write the Vertex Geometry out to a .dream3d file.
     err = simpl_helpers.WriteDREAM3DFile('vert_geom_2.dream3d', dca, True)
+    
+    # Create Cell AttributeMatrix
+    tupleDims = simpl.VectorSizeT([1, 37989])
+    am = simpl_helpers.CreateAttributeMatrix(tupleDims, 'CellAttributeMatrix', simpl.AttributeMatrix.Type.Cell)
```

Comment:
We told you the incorrect type of AttributeMatrix to create. It should be a "vertex"
``` simpl.AttributeMatrix.Type.Vertex```